metals-bench/src/main/scala/bench/PcBenchmark.scala

```diff
@@ -40,7 +70,20 @@ abstract class PcBenchmark {
     )
   }
 
-  def newPC(search: SymbolSearch = newSearch()): PresentationCompiler = {
+  def newPC(
+      version: String,
+      search: SymbolSearch = newSearch(),
+  ): PresentationCompiler = {
+    MtagsResolver.default().resolve(version) match {
+      case Some(MtagsBinaries.BuildIn) => new ScalaPresentationCompiler()
+      case Some(artifacts: MtagsBinaries.Artifacts) =>
+        embedded.presentationCompiler(artifacts, classpath)
+      case _ =>
+        throw new RuntimeException(
+          s"Forgot to release scala version: $version ?"
+        )
+
+    }
     new ScalaPresentationCompiler()
```

Comment:
Looks like the tests are only being run on scala 2 because of this

Reply:
ach, good catch!